[BUG - Develop] Built in funcparser method, `$round` is broken (doesn't return its value)

#### Describe the bug
The func parser method `$round`, implemented [here](https://github.com/evennia/evennia/blob/develop/evennia/utils/funcparser.py#L776), doesn't return its value. It is broken because of this and evaluates to the literal string `"None"`.

See following test:

```
            ("$round(2.9) apples", "3.0 apples"),
            ("$round(2.967, 1) apples", "3.0 apples"),
```

Which fail with:

```
======================================================================
FAIL: test_other_callables_22__round_2_9_apples (evennia.utils.tests.test_funcparser.TestDefaultCallables)
Test default callables [with string='$round(2.9) apples', expected='3.0 apples'].
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tmoney/code/gamedir/venv/lib/python3.9/site-packages/parameterized/parameterized.py", line 533, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/home/tmoney/code/evennia/evennia/utils/tests/test_funcparser.py", line 513, in test_other_callables
    self.assertEqual(expected, ret)
AssertionError: '3.0 apples' != 'None apples'
- 3.0 apples
+ None apples


======================================================================
FAIL: test_other_callables_23__round_2_967_1_apples (evennia.utils.tests.test_funcparser.TestDefaultCallables)
Test default callables [with string='$round(2.967, 1) apples', expected='3.0 apples'].
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tmoney/code/gamedir/venv/lib/python3.9/site-packages/parameterized/parameterized.py", line 533, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/home/tmoney/code/evennia/evennia/utils/tests/test_funcparser.py", line 513, in test_other_callables
    self.assertEqual(expected, ret)
AssertionError: '3.0 apples' != 'None apples'
- 3.0 apples
+ None apples
```

#### To Reproduce
Run tests.

#### Expected behavior
Per tests.

#### Develop-branch commit
b0f24f9975e14fddeabef06a19b5b61575957b1c

#### Additional context
Obviously this is a one line fix, it just needs to return the value it's computing. I'm filing this bug because I'm trying to write tests and my general best practice is to not modify the code-under-test at the same time, because it gets confusing. Better to just submit a skipped, failing test and come back to it.

--- a/evennia/utils/funcparser.py
+++ b/evennia/utils/funcparser.py
@@ -799,7 +799,7 @@ def funcparser_callable_round(*args, **kwargs):
     num, *significant = args
     significant = significant[0] if significant else 0
     try:
-        round(num, significant)
+        return round(num, significant)
     except Exception:
         if kwargs.get("raise_errors"):
             raise

